package.json

```diff
@@ -7,6 +7,9 @@
     "type": "git",
     "url": "https://github.com/asciidoctor/asciidoctor-docs-ui"
   },
+  "scripts": {
+    "start": "gulp preview"
```

Comment:
How important is this? Does it really add anything?

Reply:
Yes, some sort of convention. I'm working on a lot of projects and most of them are using npm scripts with predefined stage: install, build, test, start.

I think it decreases cognitive load and make it easier to get started.

Reply:
For me, gulp is the convention. (I don't particularly like npm scripts). But I understand that different people come with different perspectives, so we need to consider that. Let's handle it in a separate issue.